src/scales/scale.radialLinear.js

```diff
@@ -267,15 +267,46 @@ module.exports = function(Chart) {
 				var pointLabelPosition = scale.getPointPosition(i, outerDistance + 5);
 
 				// Keep this in loop since we may support array properties here
-				var pointLabelFontColor = valueOrDefault(pointLabelOpts.fontColor, globalDefaults.defaultFontColor);
 				ctx.font = plFont.font;
-				ctx.fillStyle = pointLabelFontColor;
-
 				var angleRadians = scale.getIndexAngle(i);
 				var angle = helpers.toDegrees(angleRadians);
 				ctx.textAlign = getTextAlignForAngle(angle);
-				adjustPointPositionForLabelHeight(angle, scale._pointLabelSizes[i], pointLabelPosition);
-				fillText(ctx, scale.pointLabels[i] || '', pointLabelPosition, plFont.size);
+
+				// Point Label Color
+                var pointLabelFontColor = null;
+				if (pointLabelOpts.fontColor) {
+					// handle scale.pointLabels[i] as string or array combined with pointLabelFontColor as string or array or none given.
+					if (typeof scale.pointLabels[i] === 'string') {
+						if (typeof pointLabelOpts.fontColor === 'string') {
```

Comment:
I would consider inverting this if and using `helpers.isArray` that way a fontColor that is not a string would still pass through (CanvasGradient, or CanvasPattern for example)